<?xml version="1.0"?>
<launch>
  <include file="$(find f1tenth_simulator)/launch/simulator.launch"/>
  
  <node name="wall_follow_navigation" pkg="lab1" type="wall_follow.py" />

	
</launch>
